index.html

```diff
@@ -2779,7 +2739,6 @@ <h4 class="headline-s-semibold centered white _2line">Photographic &amp; Non-pho
   <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.5.1.min.dc5e7f18c8.js?site=6650be52b1fb581f8c103315"
     type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
     crossorigin="anonymous"></script>
-  <script src="js/webflow.js" type="text/javascript"></script>
```

Comment:
Removing all of that seems to have broken some things, in particular: the `div-contain-carousel` thing under "Fast Encoding & Decoding" is now broken, making it stuck on the plot for lossless (there is also a plot for lossy).